app/src/main/java/com/mozilla/hackathon/kiboko/Analytics.java

```diff
@@ -16,26 +14,22 @@
 import java.util.Date;
 import java.util.List;
 
-/**
- * Created by secretrobotron on 7/2/16.
```

Comment:
These comments are place holder comments that are automatically generated by Android Studio, they can be customized in the IDE settings to automatically add a license header to each file, or do different things.

Since it is a default template and the license information is already specified in the LICENSE file, these are not needed and can be removed.